routes/api/v1/favorites.js

```diff
@@ -0,0 +1,44 @@
+var express = require('express');
+var router = express.Router();
+
+const Favorite = require('../../../pojos/favorite');
+const fetch = require('node-fetch');
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../../knexfile')[environment];
+const database = require('knex')(configuration);
+
+
+router.post('/', (request, response) => {
```

Comment:
✅ Complete basic functionality for adding a favorite. Created #18 for refactoring after all user stories have been completed.